docs/markdown_code_symlinks.py

```diff
@@ -55,9 +66,21 @@ def relative(parent_dir, child_path):
     return os.path.relpath(child_path, start=parent_dir)
 
 
-class MarkdownCodeSymlinks(transform.AutoStructify, object):
+class PrjxrayDomain(Domain):
```

Comment:
I think this should probably be a `MarkdownSymlinksDomain`?

Reply:
Done